desktop/app/goeckoh-shared.css

```
@@ -0,0 +1,308 @@
+/* =================================================================
+   GOECKOH DESIGN SYSTEM — goeckoh-shared.css
+   ================================================================= */
+
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500;600;700;800;900&family=Fira+Code:wght@400;500&display=swap');
```

Stylelint (17.14.1)

[error] Expected "url('https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500;600;700;800;900&family=Fira+Code:wght@400;500&display=swap')" to be "'https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500;600;700;800;900&family=Fira+Code:wght@400;500&display=swap'" (import-notation)

(import-notation)